### PR TITLE
fix(core): reduce unnecessary error logging when table is dropped

### PR DESCRIPTION
**Pull request by ideoma**
related to https://github.com/questdb/questdb/issues/4119

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 2 / 5 (40.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/wal/CheckWalTransactionsJob.java|1|4|25.00%|
|:large_blue_circle:|io/questdb/cairo/TxReader.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 8 / 11 (72.73%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/wal/CheckWalTransactionsJob.java|1|4|25.00%|
|:large_blue_circle:|io/questdb/cairo/TxReader.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/std/FilesFacadeImpl.java|6|6|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 8 / 13 (61.54%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/wal/CheckWalTransactionsJob.java|1|4|25.00%|
|:large_blue_circle:|io/questdb/std/FilesFacadeImpl.java|6|8|75.00%|
|:large_blue_circle:|io/questdb/cairo/TxReader.java|1|1|100.00%|

File: core/src/test/java/io/questdb/test/std/TestFilesFacadeImpl.java [hunk 1]
```diff
@@ -106,12 +106,17 @@ public void remove(LPSZ name) {
...
+    }
+
     private static synchronized boolean checkRemove(LPSZ name) {
         if (openPaths.keyIndex(name) < 0) {
             LOG.info().$("cannot remove, file is open: ").$(name).$(", fd=").$(getFdByPath(name)).$();
             // For reproducing test failures which happen on Windows only while debugging on another OS, change this to
-            // return true;
-            return Os.isWindows();
+             return false;
+//            return true;
```
Comment (bluestreak01) [inline core/src/test/java/io/questdb/test/std/TestFilesFacadeImpl.java:119]: nit: dead code

**Review by bluestreak01 — APPROVED** [review]

File: core/src/test/java/io/questdb/test/std/TestFilesFacadeImpl.java (hunk 1 above)
Comment (ideoma) [reply core/src/test/java/io/questdb/test/std/TestFilesFacadeImpl.java:119]: It's deliberate, it shows how to enable the check.

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 8 / 13 (61.54%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/wal/CheckWalTransactionsJob.java|1|4|25.00%|
|:large_blue_circle:|io/questdb/std/FilesFacadeImpl.java|6|8|75.00%|
|:large_blue_circle:|io/questdb/cairo/TxReader.java|1|1|100.00%|

**Pull request merged**